<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="singlePathRadioBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sinlgle path</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="multiPathRadioBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Multi path</string>
    </property>
   </widget>
   <widget class="QLabel" name="pathModeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Path selection mode:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PATH #1:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="path1TextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>471</width>
      <height>25</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="path2TextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>471</width>
      <height>25</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PATH #2:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openFile1Btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openFolder1Btn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openFolder2Btn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openFile2Btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadFilesBtn">
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>280</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Load files</string>
    </property>
   </widget>
   <widget class="QLabel" name="pathModeLabel_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Files extensions:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>All</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>.mem</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>.circ</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>.ods</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>.odt</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>.pdf</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_7">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>.py</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>505</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
